cloudbuild.yaml

```diff
@@ -23,7 +23,7 @@ steps:
       stack config set system-ghc --global true
       stack config set install-ghc --global false
 
-      stack install stylish-haskell hlint weeder
+      stack build stylish-haskell hlint weeder
```

Comment:
This still irks me. We should really maintain our own build image with these tools baked in.